pset-functions.js

```diff
@@ -76,6 +95,66 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let word = "function"
+word = word.toUpperCase()
+let wordScore = 0
+
+for (i=0; i<word.length; i++) {
+   wordScore = wordScore + getLetterScore(word.charAt(i))
+}
+
+console.log(wordScore)
+
+function getLetterScore(letter) {
+   let letterScore = 0
+   switch(letter) {
+      case 'A': 
+      case 'E': 
+      case 'I':
+      case 'O':
+      case 'U':
+      case 'L':
+      case 'N':
+      case 'R':
+      case 'S':
+      case 'T': 
+         letterScore = 1   
+         break
+      case 'D':
+      case 'G': 
+         letterScore = 2  
+         break
+      case 'B':
+      case 'C':
+      case 'M':
+      case 'P':
+         letterScore = 3 
+         break
+      case 'F':
+      case 'H':
+      case 'V':
+      case 'W':
+      case 'Y': 
+         letterScore = 4
+         break
+      case 'K':
+         letterScore = 5
+         break
+      case 'J':
+      case 'X': 
+         letterScore = 8
+         break
+      case 'Q':
+      case 'Z':
+         letterScore = 10
+         break
+      default:
+         break
+      }
+   return letterScore
+}
```

Comment:
Lance, lots of really cool stuff happening here! Way to get ahead of any edge cases by converting your word into Uppercase. Love how you used a switch statement to catch all possible values. I also like that you called a function before it was assigned using function declaration. For line 103, getLetterScore(word.charAt(i)) could also be written as getLetterScore(word[i])...you can search through a string's index same as you would an array :) Please find an alternate solution using the .includes method for future benefit [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L93)